README.md

```diff
@@ -310,6 +310,7 @@ ReactiveCocoa to your `Cartfile`:
 ```
 github "ReactiveCocoa/ReactiveCocoa"
 ```
+Remeber to add both ReactiveCocoa.framework and Result.framework to "Linked Frameworks and Libraries" and "copy-frameworks" Build Phases. Otherwise, it won't work.
```

Comment:
Good idea! I can see that this is not obvious at all.
I'd update the formatting a little bit:

> Make sure to add both `ReactiveCocoa.framework` and `Result.framework` to "Linked Frameworks and Libraries" and "copy-frameworks" Build Phases.